### PR TITLE
cleaned up unused variables in conftest.py and added workType to jobs_df

### PR DESCRIPTION
**Pull request by Elyrie**
new to open source so starting small to get my feet wet :)

### CODE REVIEW

**Review by pedrojunqueira — COMMENTED** [review]
@Elyrie thanks for contributing for the project. Have you ran the tests? If you added another column in the df it should break the tests. Also make sure conftest.py will still work without the request import.

**Comment by Elyrie** [comment]
Oh, thanks for the input. Just tested it with running the example, I'll try to look into it more :)

**Comment by pedrojunqueira** [comment]
> Oh, thanks for the input. Just tested it with running the example, I'll try to look into it more :)

Run the unittests with pytest. Then have a look at the change logs edit to reflect the changes and upgrade to version 0.0.3.

**Comment by Elyrie** [comment]
> > Oh, thanks for the input. Just tested it with running the example, I'll try to look into it more :)
> 
> Run the unittests with pytest. Then have a look at the change logs edit to reflect the changes and upgrade to version 0.0.3.

Thanks, I think I got it now

**Comment by Elyrie** [comment]
Wasn't sure about how to test conftest.py so I reverted the requests import deletion. I'm pretty sure the unused variable is safe to delete, but how could I test that it works? pytest runs just test_downloader.py

File: tests/conftest.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from pathlib import Path
 
 import pytest
+import requests
```
Comment (pedrojunqueira) [inline tests/conftest.py]: I tested locally and you are right. Do not need this variable to run

File: tests/conftest.py [hunk 2]
```diff
@@ -3,9 +3,9 @@
 from pathlib import Path
 
 import pytest
-import requests
 
 from job_seeker.downloader import JobSeeker
+import requests
```
Comment (pedrojunqueira) [inline tests/conftest.py]: You can remove this :)

**Review by pedrojunqueira — CHANGES_REQUESTED** [review]
Hi @Elyrie, I think it is almost there. You can remove the `requests` import in conftest and one last change. Can you change the .csv in `/example` folder with an example including the column you added?. About [conftest.py](https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session) it does not run any tests. It is just a configuration file that pytest looks and do some setup before tests are run. If you make those final changes then it is got to go. Thanks

**Comment by Elyrie** [comment]
Ok, great :) Did the last changes now

File: tests/conftest.py (hunk 2 above)
Comment (pedrojunqueira) [reply tests/conftest.py]: removed.

File: tests/conftest.py (hunk 1 above)
Comment (pedrojunqueira) [reply tests/conftest.py]: done

**Pull request merged**